nova/bigvm/manager.py

```diff
@@ -367,7 +400,10 @@ def _check_and_clean_providers(self, context, client, bigvm_providers,
                 continue
             parent_rp = vmware_providers[rp['rp']['parent_provider_uuid']]
             used_percent = parent_rp['memory_mb_used_percent']
-            if used_percent > CONF.bigvm_cluster_max_usage_percent:
+            reserved_percent = parent_rp['memory_mb_reserved_percent']
+            if used_percent > CONF.bigvm_cluster_max_usage_percent \
+                or reserved_percent > \
+                    CONF.bigvm_cluster_max_reservation_percent:
```

Comment:
Please indent lines belonging to the `if` condition once more than the body of the `if`.